bundle/jsky.app.ot/src/main/resources/resources/conf/UIInfo.xml

```diff
@@ -105,7 +105,7 @@
             type="siteQuality"
             imageKey="component.gif"
             shortDescription="This component describes the conditions under which this observation can be observed."
-            uiClassName="jsky.app.ot.gemini.editor.EdCompSiteQuality"/>
+            uiClassName="jsky.app.ot.gemini.editor.sitequality.EdCompSiteQuality"/>
```

Comment:
Yes, I just checked, and it is definitely used. `OT.initUI` calls `UIInfoXML.init`, which parses this file and uses it to create a table of editors.